### DISCUSSION TITLE
Alienfx crashing

### DISCUSSION THREAD

**Discussion by willmgf** [T-Troll/alienfx-tools#163] [General]
Guys, I`m new here in this forum but i already got the beginner`s guide.

Model:  Alienware 15R3, Intel i7-7700HQ CPU @ 2.80GHz   2.80 GHz

To Reproduce

- Start AFXControl
- All options blocked and the app crashes

Edition Windows 10 Home
Version 21H2
Installed on ‎7/‎20/‎2022
OS build 19044.1826
Windows Feature Experience Pack 120.2212.4180.0

I`m trying to use the alienfx tool to control my fans behavior.

Could you help me, please?

**Comment by T-Troll** [comment]
I need to mention - unfortunately, fan control will not work for your gear - older AW systems did not support this control methods. For now, only 13R2 supported from older models.

Crash at start looks uncommon anyway, i have 13R3 (same platform) and it works with it.
Can you please run `alienfx-cli probe=a` and share me the log?
